games-server/minecraft-server/minecraft-server-1.15.2-r1.ebuild

```diff
@@ -19,26 +19,27 @@ RDEPEND="
 	acct-group/minecraft
 	acct-user/minecraft
 	app-misc/dtach
+	sys-apps/openrc
 	|| (
 		>=virtual/jre-1.8
 		>=virtual/jdk-1.8
 	)
 "
 
-RESTRICT="bindist mirror"
+RESTRICT="bindist strip test"
```

Comment:
Why you are removing ```mirror```? According to the license, we can't distribute by the Gentoo mirrors. And why are you adding ```strip``` and ```test```? I don't see any included tests.